src/main/telemetry/telemetry.h

```diff
@@ -54,7 +54,7 @@ typedef struct telemetryConfig_s {
 
 PG_DECLARE(telemetryConfig_t, telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_MAVLINK)
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_MAVLINK | FUNCTION_TELEMETRY_IBUS)
```

Comment:
Adding TELEMETRY_IBUS to this list makes it available for sharing with all supported serial RX protocols. Since sharing will only work when shared with  iBus RX, this should not be done. Instead, sharing between iBus RX and iBus telemetry should be enabled when iBus RX is enabled. See CRSF RX / CRSF telemetry for how it's done.